### PR TITLE
Fix #2404 Longpress works correctly for selecting multiple photos in allmedias.

### LINKED ISSUE #2404: LongPress issue in AllPhotos Activity
**Actual Behaviour**

LongPressing on 2nd photo after selecting first photo doesn't select all photos in between in AllPhotos Activity.

**Expected Behaviour**

All the photos between two photos should get selected as they do in other activities.

**Would you like to work on the issue?**

YES

### PR DESCRIPTION
**Pull request by angmas1**
Fixed #2404 

Changes: Files changes: LFMainActivity.java. Fixed bug where LongPressing one photo after selecting one photo in AllPhotos Activity didn't automatically select all photos in between.

### CODE REVIEW

File: app/src/main/java/org/fossasia/phimpme/gallery/activities/LFMainActivity.java [hunk 1]
```diff
@@ -781,7 +781,7 @@ public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         Log.e("TAG", "lfmain");
         ButterKnife.bind(this);
-      
+
```
Comment (yashk2000) [inline app/src/main/java/org/fossasia/phimpme/gallery/activities/LFMainActivity.java]: Please remove the extra line.

**Comment by yashk2000** [comment]
@angmas1 please add screenshots or GIFs of the changes you make so that it is easy to see what change has been made. Please follow this for every pull request you make,

File: app/src/main/java/org/fossasia/phimpme/gallery/activities/LFMainActivity.java (hunk 1 above)
Comment (angmas1) [reply app/src/main/java/org/fossasia/phimpme/gallery/activities/LFMainActivity.java]: ok

**Comment by angmas1** [comment]
ezgif com-video-to-gif

**Comment by angmas1** [comment]
@abishekvashok please review

**Comment by mariobehling** [comment]
Please resolve conflicts.

**Comment by angmas1** [comment]
@mariobehling resolved please merge

**Pull request merged**